test/Mono.Linker.Tests.Cases/Inheritance.Interfaces/StaticInterfaceMethods/Dependencies/TypeHierarchyBasic.il

```diff
@@ -3,45 +3,17 @@
 // See the LICENSE file in the project root for more information.
 
 .assembly extern mscorlib { }
-.assembly extern System.Runtime {}
 
 .assembly 'library' { }
 
-.class interface public auto ansi abstract IMyInterface
-{
-  .method public hidebysig newslot abstract virtual 
-          instance void  Frob() cil managed
-  {
-  } // end of method InterfaceScenario1::Method
-
-}
-
-.class public auto ansi MyClass
-	extends [System.Runtime]System.Object
-	implements IMyInterface
-{
-  .method public hidebysig specialname rtspecialname 
-          instance void  .ctor() cil managed
-  {
-    ret
-  } // end of method MyClass::.ctor
-
-  .method public hidebysig newslot virtual final 
-          instance void  Frob() cil managed
-	{
-        call void TestEntrypoint::Test()
-		ret
-	}
-}
-
 .class interface public abstract auto ansi InterfaceScenario1
 {
   .method public newslot virtual abstract static int32 Method() cil managed noinlining
   {
   } // end of method Method
 } // end of class InterfaceScenario1
 .class public auto ansi BaseScenario1
-       extends [System.Runtime]System.Object
+       extends [mscorlib]System.Object
```

Comment:
Why are we targeting mscorlib? These tests are never expected to work on anything but .NET 6+, so there's no reason to use mscorlib.

Reply:
When referencing System.Runtime the test would generate errors stating that the assembly System.Runtime couldn't be found. I guess in part is inexperience with the IL testing in linker. Not sure if there is something else that needs to be made in the testing infra to be able to reference System.Runtime instead of mscorlib

Reply:
That's because there's `.assembly extern mscorlib { }` above. If you dump simple .NET Core app it will have `.assembly extern System.Runtime { }` instead.

In IL you have to declare assembly references, and only the declared ones will be "usable".

Reply:
That line existed in the test file and was the one generating a roslyn error

Reply:
hmm - Roslyn error? OK - I would have to repro this I guess, but the current state is not great. (I assume the tests in runtime repo are not targeting mscorlib, right?)

Reply:
I experimented a little bit more with it
If you only reference `.assembly extern System.Runtime { }` and then use it in the test, you will get the following failure:
```
System.AggregateException : One or more errors occurred. (Roslyn compilation errors: (20,5): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.
    )
      ----> System.Exception : Roslyn compilation errors: (20,5): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.
```
But if you put a version and publickeytoken
```
.assembly extern System.Runtime
{
  // I took those values from a dotnet app that I build
  .publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A)
  .ver 6:0:0:0
}
```
Then the test will pass